tox.ini

```diff
@@ -17,7 +17,11 @@ commands = nosetests --with-coverage --cover-package=pylxd pylxd
 commands = flake8 --ignore=E123,E125
 
 [testenv:integration]
-commands = nosetests integration
+whitelist_externals =
+    env
+commands =
+    env HTTP_PROXY='http://127.0.0.42' nosetests integration.test_proxies
```

Comment:
I appreciate that this validates the fix, but I don't think it's needed. If the fix is in a dependency, we don't need to test the dependency. I don't think this part of the patch is needed.